playbooks/roles/ocp-post-install/tasks/main.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Can we create individual task file for customizations and include them in main.yaml ?
For example this can be kernel.yaml. Likewise when we have others we can just create separate files and keep adding them to main.yaml. This way disabling a particular customization also will become easier.